src/ui/components/Rating/index.js

```diff
@@ -96,7 +96,7 @@ export class RatingBase extends React.Component {
     if (rating) {
       description = i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5'),
         { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
-    } else {
+    } else if (!readOnly) {
```

Comment:
Should we be displaying something other than nothing if it's `readOnly` and there are no ratings? Something like `This add-on has not been rated yet`? That seems to be suggested in issue #4706.

Reply:
Sure, we can do that. I'll make the change.